pkg/rid/application/application_test.go

```diff
@@ -58,6 +59,10 @@ func setUpStore(ctx context.Context, t *testing.T, logger *zap.Logger) (store.St
 			},
 		}, func() {}
 	}
+	if !(strings.Contains(*storeURI, "rid") || strings.Contains(*storeURI, "scd")) {
+		*storeURI = strings.Replace(*storeURI, "?sslmode", "/rid?sslmode", 1)
```

Comment:
Is this the replacement we want when storeURI contains "scd"?  Shouldn't it be "/scd?sslmod" in that case rather than "?sslmode"?  Or possibly no replacement in that case?